### ISSUE TITLE
Add a "released" signal to Slider

### ISSUE THREAD

**Issue by Calinou** [godotengine/godot#7056]
Sliders should have a way to trigger something only after they have been released, not continuously while they are being dragged.

An example case would be for a computationally intensive operation that can't be done on every frame.

I tested the `released()` signal but it did not seem to work, at least with mouse dragging.

**Comment by nerzhul** [comment]
A usage case we could have in our project is to a register a setting to file attach to this slider directly after releasing the slider.

**Comment by vnen** [comment]
Similar to #6007.

**Comment by bojidar-bg** [comment]
I'd say, complete duplicate of #6007. Thus closing.
